doc/reference/reference_lua/box_ctl/make_bootstrap_leader.rst

```diff
@@ -0,0 +1,20 @@
+.. _box_ctl-make_bootstrap_leader:
+
+box.ctl.make_bootstrap_leader()
+===============================
+
+.. function:: make_bootstrap_leader()
+
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Make the instance a bootstrap leader of a replica set.
```

Comment:
I'm not sure how the new configuration works, but it's worth mentioning that `box.ctl.make_bootstrap_leader()` should be called after `box.cfg()` call is issued. I suppose this is the only way to do this with the new configuration anyway.
The node should be listening on a console `box.cfg.listen` or `require'console'.listen()` while running `box.cfg` in order for the user to be able to send the `box.ctl.make_bootstrap_leader()` command. Otherwise there will be a deadlock: `box.cfg()` on the desired bootstrap leader doesn't end until `box.ctl.make_bootstrap_leader()` is issued.

@Totktonada, can you confirm that everything works fine in this case?

Reply:
I performed several experiments around `supervised` bootstrap mode with config in September. I found no easy way to use it.

Config starts the admin interface after starting the database (`console.listen()` goes after `box.cfg()`). So, we have no way to give a server commands before the bootstrap.

Moreover, the server can't perform some user specified code before `box.cfg()` to, say, pool some bootstrap coordinator or listen on a separate command socket.

This looks like a true deadlock.

I would discuss f2f what is better to do with `replication.bootstrap_strategy: supervised` in the declarative configuration.

Reply:
We discussed it with Sergey and agreed that the admin console should be started before `box.cfg()`. I've added a subtask to https://github.com/tarantool/tarantool/issues/8862.

Reply:
So, am I correct that these are internal details for a declarative configuration approach and I shouldn't add anything to the `make_bootstrap_leader()` description?

Reply:
> So, am I correct that these are internal details for a declarative configuration approach and I shouldn't add anything to the make_bootstrap_leader() description?

Current description looks good to me, I think we can continue with this PR without waiting for updates in declarative config internals, if @Totktonada doesn't object.